bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -0,0 +1,124 @@
+| revolution | Thing      | A Freebox Revolution server.                          |
+| player     | Thing      | The TV player equipment (e.g. Devialet).              |
+| landline   | Thing      | The phone wired to the Freebox Server.                |
+| host       | Thing      | A network device on the local network.                |
+| wifihost   | Thing      | A wifi networked device on the local network.         |
+| vm         | Thing      | A virtual machine hosted on the server (>= Delta).    |
+| repeater   | Thing      | A Free wifi repeater.                                 |
+
+## Discovery
+
+The API bridge is discovered automatically through mDNS in the local network.
+After the bridge is discovered and available to openHAB, the binding will automatically discover phone, network devices available on the local network.
+Note that the discovered thing will be setup to use only HTTP API (and not HTTPS) because only an IP can be automatically determined while a domain name is required to use HTTPS with a valid certificate.
+
+## Binding configuration
+
+The binding itself is not configurable.
+
+## Thing Configuration
+
+### Api bridge
+
+| Parameter Label          | Parameter ID      | Description                                            | Required | Default              |
+|--------------------------|-------------------|--------------------------------------------------------|----------|----------------------|
+| Freebox Server Address   | apiDomain         | The domain to use in place of hardcoded Freebox ip     | No       | mafreebox.freebox.fr |
+| Application Token        | appToken          | Token generated by the Freebox Server.                 | Yes      |                      |
+| Network Device Discovery | discoverNetDevice | Enable the discovery of network device things.         | No       | false                |
+| HTTPS Available          | httpsAvailable    | Tells if https has been configured on the Freebox      | No       | false                |
+| HTTPS port               | httpsPort         | Port to use for remote https access to the Freebox Api | No       | false                |
+
+If the parameter *apiDomain* is not set, the binding will use the default address used by Free to access your Freebox Server (mafreebox.freebox.fr).
+The bridge thing will initialize only if a valid application token (parameter *appToken*) is filled.
+
+### Server : Revolution or Delta
+
+The *revolution* or *delta* bridge thing requires the following configuration parameters:
+
+| Parameter Label  | Parameter ID    | Description                                                              | Required | Default |
+|------------------|-----------------|--------------------------------------------------------------------------|----------|---------|
+| Refresh Interval | refreshInterval | The refresh interval (seconds) which is used to poll the Freebox Server. | Yes      | 30      |

```

Comment:
I think I may have a misunderstanding : having a default value on a given configuration element does not change the fact that it is required or not. If you've got a default value and delete it it will not work, so it should remain required. No ?

Reply:
If you "delete" it, the default value will then be considered and provided.
To be checked to be 100% sure.